src/aerie_cli/aerie_host.py

```diff
@@ -18,15 +25,12 @@ def process_gateway_response(resp: requests.Response) -> dict:
         dict: Contents of response JSON
     """
     if not resp.ok:
-        raise RuntimeError(f"Bad response from Aerie Gateway.")
+        raise RuntimeError("Bad response from Aerie Gateway")
 
     try:
         resp_json = resp.json()
     except requests.exceptions.JSONDecodeError:
-        raise RuntimeError(f"Failed to get response JSON")
-
-    if "success" in resp_json.keys() and not resp_json["success"]:
-        raise RuntimeError(f"Aerie Gateway request was not successful")
```

Comment:
You ought to move this check out to when you're hitting the `login` endpoint.

Reply:
Thanks - I'm honestly not sure why I removed this. I wrote most of this months ago, so give it another look...

Reply:
The `version` endpoint doesn't return `success`

Reply:
Maybe why I did it originally, but it still only checked if the return value included the "success" key, so this branch isn't taken for a `get` request's response. Putting it back in and I'll see if anything breaks in the integration tests...

Reply:
Oh huh, you're right. Yeah, if it doesn't break anything, I'd say revert this change.